### PR TITLE
Added consumes calls to StripCPEESProducer

### PR DESCRIPTION
**Pull request by Dr15Jones**
#### PR description:

-Switched to using ESGetTokens
-Removed 3 get calls to data which were never used
-Consolidated header and source into one file

#### PR validation:

Code compiles.
The changes are technical and should not change results.

### CODE REVIEW

**Comment by Dr15Jones** [comment]
please test

File: RecoLocalTracker/SiStripRecHitConverter/plugins/StripCPEESProducer.cc [hunk 1]
```diff
@@ -12,6 +19,25 @@
 #include "CondFormats/SiStripObjects/interface/SiStripBadStrip.h"
 #include "FWCore/Framework/interface/ESHandle.h"
```
Comment (perrotta) [inline RecoLocalTracker/SiStripRecHitConverter/plugins/StripCPEESProducer.cc]: The following three headers can be removed now:
```C++
#include "CondFormats/SiStripObjects/interface/SiStripNoises.h"
#include "CondFormats/SiStripObjects/interface/SiStripApvGain.h"
#include "CondFormats/SiStripObjects/interface/SiStripBadStrip.h"
```

And maybe you can profit and move the FWCore one on top, together with the other include's of FWCore headers

Comment (Dr15Jones) [reply RecoLocalTracker/SiStripRecHitConverter/plugins/StripCPEESProducer.cc]: Done

**Comment by Dr15Jones** [comment]
please test

**Comment by perrotta** [comment]
+1
- Technical
- Jenkins tests pass

**Comment by fabiocos** [comment]
+1

**Comment by makortel** [comment]
part of #26748

**Pull request merged**